jax/_src/lax/lax.py

```diff
@@ -6434,23 +6434,31 @@ def _infeed_translation_rule(c, token, *, shapes, partitions):
 infeed_p.def_abstract_eval(_infeed_abstract_eval)
 xla.translations[infeed_p] = _infeed_translation_rule
 
-def outfeed(token, xs):
+def outfeed(token, xs, partitions = None):
   """Outfeeds value `xs` to the host. Experimental.
 
   `token` is used to sequence infeed and outfeed effects.
```

Comment:
Maybe extend the docstring with how `partitions` works?

Reply:
Done